frontend/src/router/index.js

```diff
@@ -170,7 +170,7 @@ const router = createRouter({
 router.beforeEach((to, _from, next) => {
   // this section is to set page title in vue store
   const appStore = useAppStore()
-  if (to && to.meta) {
+  if (to && to.meta.pageTitle) {
```

Comment:
I think it might make more sense to leave as it but ensure that Login route has a pageTitle defined.

Reply:
Note: adding page title to the check removes console warning... 
![image](https://github.com/bcgov/ecc-ofm/assets/61481459/99589dba-396b-4f5c-b053-d4992d476489)